.travis.yml

```diff
@@ -4,13 +4,11 @@ services:
   - docker
 
 install:
-  - docker build --rm -t buildenv docker/
+  - docker build --rm -t manylinux1_i686 -f docker/Dockerfile-i686 docker/
+  - docker build --rm -t manylinux1_x86_64 -f docker/Dockerfile-x86_64 docker/
 
 script:
   >
     set -ex;
-    docker run -t --rm -v $(pwd):/io buildenv sh -c 'set -e; for INTERP in /opt/*/bin/python; do $INTERP /io/manylinux1-check.py; done';
-    PACKAGES=(numpy);
-    for pkg in "${PACKAGES[@]}"; do
-      docker run -t --rm -v `pwd`:/io buildenv /opt/3.5/bin/pip install --no-binary :all: ${pkg} -vv;
-    done
+    docker run -t --rm -v $(pwd):/io manylinux1_x86_64 sh -c 'set -e; for INTERP in /opt/*/bin/python; do $INTERP /io/manylinux1-check.py; done';
```

Comment:
Will this pick up the symlinks?  I mean, are we testing /opt/2.7.11 as well as the identical /opt/2.7 ?